consts.php

```diff
@@ -41,6 +41,11 @@
         '8.2',
         '8.3',
     ],
+    '6' => [
+        '8.1',
+        '8.2',
+        '8.3',
```

Comment:
We'll only support `8.3` to start with - but I guess we can remove this when we bump the PHP dep so no change required for this just yet, just thinking aloud.